.ci/release_template.md

```diff
@@ -53,12 +56,27 @@ Use these label badges for highlighting important and key changes:
 --REPLACE-WITH-GENERATED-LIST--
  -->
 
+ - ⚠️ **Important:**
+    - --REPLACE-WITH-HIGHLIGHTED-IMPORTANT-CHANGES--
```

Comment:
the --REPLACE-- placeholders are for the scripts, they won't be shown to humans, this should just be put in a human facing comment.

Reply:
Makes sense. I saw you changed the CI stuff and scripts, but did not look in detail yet. 🙈 
Will rephrase.

Reply:
The script here is what is actually using this template: https://github.com/Cockatrice/Cockatrice/blob/master/.ci/prep_release.sh#L85

Reply:
Thanks for linking.

Reply:
I mean, it's the top line in the template for a reason.....